cmd/algorelay/eb/eb.go

```diff
@@ -18,9 +18,9 @@ package eb
 
 // Relay represents the configuration data necessary for a single Relay
 type Relay struct {
-	ID                    int64  // db key injected when loaded
```

Comment:
this isn't wrong, but it's irrelevant to the core of the change around checksum addresses so I'd just like to see a more minimal change without reformatting of random other parts of the code. also in cmd/algorelay/relayCmd.go

Reply:
These changes are form gofmt I think. just pushed a change to these two - didn't spot any others changed in this way. 

V/r,